plugins/osdn/vnids.go

```diff
@@ -34,7 +69,7 @@ func (oc *OsdnController) GetVNID(name string) (uint, error) {
 		return id, nil
 	}
 	// In case of error, return some value which is not a valid VNID
-	return MaxVNID + 1, fmt.Errorf("Failed to find netid for namespace: %s in vnid map", name)
+	return netid.MaxVNID + 1, fmt.Errorf("Failed to find netid for namespace: %s in vnid map", name)
```

Comment:
returned uint shouldn't matter in an error case, use 0 rather than adding 1 to a number that could cause an overflow, depending on the constant value